src/index.js

```diff
@@ -72,30 +72,37 @@ function glamorous(comp, {rootEl, displayName, forwardProps = []} = {}) {
       }
 
       render() {
-        const {className, innerRef, ...rest} = this.props
-        const {toForward, cssOverrides} = splitProps(rest, GlamorousComponent)
+        // in this function, we're willing to sacrafice a little on
+        // readability to get better performance because it actually
+        // matters.
+        const props = this.props
+        const {toForward, cssOverrides} = splitProps(props, GlamorousComponent)
+        // freeze the theme object in dev mode
```

Comment:
Just a handy thing for dev mode so we don't have to make a copy of the object in production mode.